plugins/generator-1.16.4/forge-1.16.4/procedures/world_switch_world.java.ftl

```diff
@@ -0,0 +1,17 @@
+if (world instanceof ServerWorld) {
+    IWorld _worldorig = world;
+
+    <#if field$dimension=="Surface">
+        world = ((ServerWorld) world).getServer().getWorld(DimensionType.OVERWORLD);
+    <#elseif field$dimension=="Nether">
+        world = ((ServerWorld) world).getServer().getWorld(DimensionType.THE_NETHER);
+    <#elseif field$dimension=="End">
+        world = ((ServerWorld) world).getServer().getWorld(DimensionType.THE_END);
+    <#else>
+        world = ((ServerWorld) world).getServer().getWorld(${(field$dimension.toString().replace("CUSTOM:", ""))}Dimension.type);

```

Comment:
This will not work, there is no loger .type in dimensions. Same problem is reported in #636 for other elements

Reply:
What do I do then? I delete the procedure until you fix the bug?

Reply:
No, it is not a bug, but wrong access usage. You need to load dimension type from registry now, like for example in entity_switch_dimension. 'DimensionType' is no longer used too.

Reply:
I have fixed said bug. Still I recommend you use entity_switch_dimension for reference, not the fix from #636, as entity_switch_dimension is more related to this procedure block.

Reply:
Procedure block removed from PR.